specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2025-07-01/examples/autoImportJobs_CreateOrUpdate.json

```diff
@@ -0,0 +1,132 @@
+      },
+      "location": "eastus",
+      "properties": {
+        "autoImportPrefixes": [
+          "/"
+        ],
+        "conflictResolutionMode": "Skip",
+        "enableDeletions": false,
+        "maximumErrors": 0,
+        "adminStatus": "Enable"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "tags": {
+          "Dept": "ContosoAds"
+        },
+        "name": "autojob1",
+        "location": "eastus",
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/scgroup/providers/Microsoft.StorageCache/amlfilesystems/fs1/autoImportJobs/autojob1",
+        "type": "Microsoft.StorageCache/amlFilesystem/autoImportJob",
+        "properties": {
+          "adminStatus": "Enable",
+          "provisioningState": "Succeeded",
+          "autoImportPrefixes": [
+            "/"
+          ],
+          "conflictResolutionMode": "Skip",
+          "enableDeletions": false,
+          "maximumErrors": 0,
+          "status": {
+            "state": "InProgress",
+            "scanStartTime": "2024-03-21T17:25:43.511Z",
+            "scanEndTime": "2024-03-21T18:25:43.511Z",
+            "totalBlobsWalked": 1000000,
+            "rateOfBlobWalk": 10000,
+            "totalBlobsImported": 900000,
+            "rateOfBlobImport": 4000,
+            "importedFiles": 8000,
+            "importedDirectories": 1000,
+            "importedSymlinks": 1000,
+            "preexistingFiles": 800000,
+            "preexistingDirectories": 80000,
+            "preexistingSymlinks": 10000,
+            "totalErrors": 10,
+            "totalConflicts": 5,
+            "blobSyncEvents": {
+              "importedFiles": 500,
+              "importedDirectories": 100,
+              "importedSymlinks": 50,
+              "preexistingFiles": 10000,
+              "preexistingDirectories": 1000,
+              "preexistingSymlinks": 200,
+              "totalBlobsImported": 11850,
+              "rateOfBlobImport": 120,
+              "totalErrors": 3,
+              "totalConflicts": 1,
+              "deletions": 20,
+              "lastChangeFeedEventConsumedTime": "2024-03-21T18:35:43.511Z",
+              "lastTimeFullySynchronized": "2024-03-21T18:25:43.511Z"
+            }
+          }
+        }
+      }
+    },
+    "201": {
+      "body": {
+        "tags": {
+          "Dept": "ContosoAds"
+        },
+        "name": "autojob1",
+        "location": "eastus",
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/scgroup/providers/Microsoft.StorageCache/amlfilesystems/fs1/autoImportJobs/autojob1",
+        "type": "Microsoft.StorageCache/amlFilesystem/autoImportJob",
+        "properties": {
+          "adminStatus": "Enable",
+          "provisioningState": "Succeeded",
+          "autoImportPrefixes": [
+            "/"
+          ],
+          "conflictResolutionMode": "Skip",
+          "enableDeletions": false,
+          "maximumErrors": 0,
```

Comment:
>maximumErrors [](http://example.com/codeflow?start=11&length=13)

possible point of usability confusion:

does maximumErrors=0 mean no retries, or does maximumErrors=1 no retries?

If they mean the same, why have two different values?